ROADMAP.md

```diff
@@ -27,22 +27,30 @@ the library into 3 pieces.
 - ✅ `libvcx` - adds memory management and C bindings on top of `aries-vcx` - making it consumable
    on Android, iOS and any programming language.
 
-# Phase 2 - In progress 🚧
-- 🚧 Migration from `libindy` to its fork [vdr-tools](https://gitlab.com/evernym/verity/vdr-tools).
-- 🚧 Concise `aries-vcx` crate API and start publishing on crates.io
-- 🚧 Implement testing backchannel for aries-vcx. [WIP](https://github.com/hyperledger/aries-agent-test-harness/pull/243)
-- Support for public DID-based connection invitations
-- Support for [out-of-band protocol](https://github.com/hyperledger/aries-rfcs/tree/master/features/0434-outofband)
+# Phase 2 - Done ✅
+- ✅ Migration from `libindy` to its fork [vdr-tools](https://gitlab.com/evernym/verity/vdr-tools).
+- ✅ Removal of blocking calls to `vdr-tools`
+- ✅ Support for public DID-based connection invitations
+- ✅ Support for [out-of-band protocol](https://github.com/hyperledger/aries-rfcs/tree/master/features/0434-outofband)
+- ✅ Implement testing backchannel for aries-vcx. [Test runs](https://github.com/hyperledger/aries-agent-test-harness/actions)
+- ✅ Removal of global state from `aries-vcx`
+- ✅ Support wallet multi-tenancy on aries-vcx level
 
-# Future work 
+# Phase 2 - In progress 🚧 / Planned 
+- 🚧 Update C API / public-agents API to enable for receiving messages from public directly, without mediator intermediary
```

Comment:
Update public agents' API to enable for receiving messages directly, without a mediator intermediary.